fvtest/porttest/omrtimeTest.cpp

```diff
@@ -314,7 +314,7 @@ TEST(PortTimeTest, time_test3)
 			/*change of millis*/
 			time = omrtime_current_time_millis();
 			oldTime = time;
-			for (j = 0; oldTime == time; j++) {
+			while(oldTime == time) {
```

Comment:
there needs to be a space here, as per the coding standard: https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#formatting-conditions
```suggestion
			while (oldTime == time) {
```